fhevm/tee_interpreter.go

```diff
@@ -25,7 +25,7 @@ func doOp(
 ) ([]byte, error) {
 	logger := environment.GetLogger()
 
-	lp, rp, lhs, rhs, err := extract2Operands(op, environment, input, runSpan)
+	isScalar, lp, rp, lhs, rhs, err := extract2Operands(op, environment, input, runSpan)
```

Comment:
could you change extract2Operands signature to have:

```suggestion
	lp, rp, lhs, rhs, isScalar, err := extract2Operands(op, environment, input, runSpan)
```

Reply:
Okay